src/libraries/IOwnable.sol

```diff
@@ -0,0 +1,7 @@
+pragma solidity ^0.4.13;
+
+
+contract IOwnable {
+    function getOwner() public constant returns (address);
+    function transferOwnership(address newOwner) public returns (bool);
```

Comment:
Are modifiers not required for interface function declarations (such as `onlyOwner` in this case)?

Reply:
Correct, they are not required and in fact not allowed in this case since this interface doesn't know what the modifiers mean.  The modifiers are an implementation detail.  In theory, we could make the various modifiers available in interfaces, at which point we could apply them as interfaces.  I'm on the fence as to whether that is a good thing or not.